### ISSUE TITLE
Double ids throwing wrong warning

### ISSUE THREAD

**Issue by PalleZingmark** [rossPatton/stylint#210]
We´re using `v1.2.3`.
```css
div#my-id #another-id {
   // ...
}
```
`-> Warning: hexidecimal color should be a variable`

Although this code isn't optimal, this shouldn't throw a warning for hexidecimal color.

**Comment by nayaabkhan** [comment]
@PalleZingmark Cannot reproduce this one on the latest. Could you update stylint and try again?

**Comment by PalleZingmark** [comment]
@nayaabkhan Confirmed, cannot reproduce this one on the latest.
